tools/greenbone-nvt-sync.in

```diff
@@ -367,7 +367,7 @@ is_feed_current () {
 }
 
 do_wget_community_feed () {
-  TMP_NVT="$SYNC_TMP_DIR/openvas-feed-`date +%F`-$$.tar.bz2"
+  TMP_NVT="$SYNC_TMP_DIR/gcf-`date +%F`-$$.tar.bz2"
```

Comment:
@janowagner @bjoernricks 

Either use:

```
gcf-`date +%F`-$$.tar.bz2"
```

for both (wget and curl) or:

```
gcf-feed-`date +%F`-$$.tar.bz2"
```

not a mixture of both.